.vscode/launch.json

```diff
@@ -0,0 +1,57 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
```

Comment:
Do we know that the vscode JSON parser is json5 compliant?

Reply:
Don't know for sure, this comment was generated by VSCode's own templates though

Reply:
If it works then great!